pkg/webhook/policy.go

```diff
@@ -68,7 +69,10 @@ import (
 // https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#response
 const httpStatusWarning = 299
 
-var maxServingThreads = flag.Int("max-serving-threads", -1, "cap the number of threads handling non-trivial requests, -1 caps the number of threads to GOMAXPROCS. Defaults to -1.")
+var (
+	ValidateTemplateRego = flag.Bool("validate-template-rego", true, "validate rego code for constraints template. Defaults to true. To be removed in 3.16. Use gator to validate in shift left manner to avoid impact with this behavior change. Not to be used when `experimental-enable-k8s-native-validation` is set to true.")
```

Comment:
```suggestion
	ValidateTemplateRego = flag.Bool("validate-template-rego", true, "validate Rego code for constraint templates. Defaults to true. This flag will be removed in Gatekeeper v3.16 and cannot be used if experimental-enable-k8s-native-validation flag is set. Use Gator to validate in shift left manner to avoid impact with this behavior change.)
```